routes/apis.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var express = require('express');
+var where = require("lodash");
```

Comment:
No need to pull in `lodash` for `filter`; in node.js, `Array.prototype.filter()` is already defined. Plus you never added `lodash` to `package.json` (and please don't).